share/locale/de_DE/LC_MESSAGES/eduVPN.po

```diff
@@ -0,0 +1,248 @@
+# German translations for python-eduvpn-client package.
+# Copyright (C) 2020, The Commons Conservancy eduVPN Programme
+# This file is distributed under the same license as the python-eduvpn-client package.
+# Ralf Becker <beckerr@hochschule-trier.de>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: python-eduvpn-client-1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-21 22:38+0100\n"
+"PO-Revision-Date: 2020-05-16 00:21+0200\n"
+"Last-Translator: Ralf Becker <beckerr@hochschule-trier.de>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: ../eduvpn/builder/mainwindow.ui:1395
```

Comment:
A number of these strings from the `.ui` file aren't actually used, so they should probably removed there instead of requiring all translators to go over them.

Reply:
I mean removing them from from the `mainwindow.ui` file. I assume remove the `translatable="yes"` property there will also work.

Reply:
Yes, I think that would work